.travis.yml

```diff
@@ -3,7 +3,6 @@ sudo: false
 language: ruby
 cache: bundler
 rvm:
-  - 2.3
```

Comment:
Are we discreetly loosing support for 2.3 or not testing it anymore?

Reply:
Since we started using `T::Struct` in `Config`, we lost 2.3 support. `sorbet-runtime` seems to be using `String#match?` in their implementation which is not supported in 2.3, so I am dropping it, since anybody using `sorbet-runtime` cannot be running 2.3 anyway.